lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +62,19 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+    listed_combinations = []
+    combination_frequencies = {}
+    for word in word_frequencies:
+        for i, token in enumerate(word):
+            if (i + 1) < len(word):
+                listed_combinations.append((token, word[i + 1]))
+    for combination in listed_combinations:
```

Comment:
why not add them to the dictionary immediately in the first cycle?

Reply:
Done